source/use_cases/aws-restaurant-management-demo/README.md

```diff
@@ -42,17 +64,33 @@ This demo implements three REST APIs using AWS Gateway, one for each user type.
   - `orderId` - the unique order id.
 
 ### Kitchen staff
-- `/list-open-orders` - GET - lists all orders that have a status of 'OPEN'.
+- `/get-open-orders` - GET - lists all orders that have a status of 'OPEN'.
```

Comment:
For the same reason you probably changed list to get, I like "retrieves" better than "lists" - this is the API call, there is no UI that presents them in a list to users (that happens elsewhere).